advanced/is-pattern-1/values.yaml

```diff
@@ -32,6 +32,9 @@ wso2:
     # Exported location on external NFS Server to be mounted at <IS_HOME>/repository/deployment/server/userstores
     sharedUserstores: ""
 
+  # The host for the ingress and HOST_NAME environment variable
+  host: "localhost"
```

Comment:
@rgauss also shall we move this parameter to section `wso2.deployment.wso2is` section as this hostname directly refers to WSO2 Identity Service when exposed to outside? WDYT?

Reply:
@chirangaalwis, I was under the impression that the `deployment` section was specific to the Kubernetes `Deployment` resource.

Since this affects both the ingress and the container's `HOST_NAME` environment variable it seemed to make sense to put it at a higher level, but if the `deployment` section is more general it may make more sense to have it there.

It sounds like that's the case so I'll move it.

Reply:
@chirangaalwis, moved to `wso2.deployment.wso2is.host`.